src/createAnimatedComponent.tsx

```diff
@@ -479,6 +595,17 @@ export default function createAnimatedComponent(
                 };
               }
               return this.initialStyle;
+            } else if (hasInlineStyles(style)) {
+              const newStyle: StyleProps = {};
+              for (const [key, styleValue] of Object.entries(style)) {
+                if (
+                  styleValue.value === undefined &&
+                  !(key === 'transform' && isInlineStyleTransform(styleValue))
+                ) {
+                  newStyle[key] = styleValue;
+                }
```

Comment:
if it is shared value don't we still include it in `newStyle` ? Much like we do with `useAnimatedStyle` where we just take the initial value, so here maybe we should do `newStyle[key] = styleValue.value`